website/content/en/docs/setup/installation/package-managers/helm.md

```diff
@@ -4,33 +4,60 @@ short: Helm
 weight: 3
 ---
 
-[Helm] is a package manager for Kubernetes that facilitates the deployment and management of applications and services on Kubernetes clusters. This page covers installing and managing Vector through the Helm package repository.
-
-## Installation
+[Helm] is a package manager for Kubernetes that facilitates the deployment and management of applications and services on Kubernetes clusters. This page covers installing and managing the Vector Agent and the Vector Aggregator through the Helm package repository. The Agent and the Aggregator are added by adding the Vector Helm chart but are installed onto each node and configured separately.
 
 {{< warning title="Aggregator role in private beta" >}}
 Helm support for the [aggregator] role is currently in private beta. We're currently seeking beta testers. If interested, please [join our chat][chat] and let us know.
 
-As an alternative, you can still manually deploy Vector in the aggregator role. Instructions throughout this doc will be for the [agent] role only.
-
 [agent]: /docs/setup/deployment/roles/#agent
 [aggregator]: /docs/setup/deployment/roles/#aggregator
 [chat]: https://chat.vector.dev
 {{< /warning >}}
 
-Add the Vector repo:
+## Adding the Helm repo
+
+If you haven't already, start by adding the Vector repo:
 
 ```shell
-helm repo add vector https://helm.vector.dev
+helm repo add timberio https://packages.timber.io/helm/latest
+helm repo update
 ```
 
-Check available Helm chart configuration options:
+## Agent
+
+The Vector [Agent] lets you collect data from your [sources] and then deliver it to a variety of destinations with [sinks].
+
+### Installing
+
+Once you add the Vector Helm repo, install the Vector Agent to each node:
 
 ```shell
-helm show values vector/vector-agent
+helm install vector timberio/vector-agent \
+  --namespace vector \
+  --create-namespace \
+  --values values.yaml
```

Comment:
One note is the user may not have a local `values.yaml` until we have them generate the example on ln 63.

`helm install` without the `-f`/`--values` option would use our packaged configuration which is valid, but doesn't do much today.

Reply:
okay, that makes a lot more sense. I will swap them in both!